serverless/messaging/how-to/create-manage-queues.mdx

```diff
@@ -0,0 +1,117 @@
+---
+meta:
+  title: How to create and manage SQS/SNS queues
+  description: This page explains how to create and manage SQS/SNS queues
+content:
+  h1: How to create and manage SQS/SNS queues
+  paragraph: This page explains how to create and manage SQS/SNS queues
+tags: messaging queuing queues sns sqs fifo
+categories:
+  - serverless 
+dates:
+  validation: 2023-01-04
+  posted: 2022-08-11 
+---
+
+This page shows how to create and manage [queues](/serverless/messaging/concepts/#queue) in a [Messaging and Queuing SQS/SNS namespace](/serverless/messaging/concepts/#namespace). Note that queues cannot be created via the console or Scaleway API for a NATS namespace. 
```

Comment:
@amal-krm @grom-42 @albinou

Reply:
I would prefer to leave this here since we specify the console and Scaleway API, neither of which can be used to create NATS queues. I think it's good to make it clear that though we have this ability for SQS/SNS queue creation in the console, it doesn't exist in parallel for NATS.

Reply:
> Note that queues cannot be created via the console 

Normally we can create a queue in the console now

Reply:
But not for NATS @amal-krm ?

Reply:
Right, not for NATS